codebuild/spec/buildspec_fuzz_batch.yml

```diff
@@ -0,0 +1,50 @@
+---
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+version: 0.2
+
+# This buildspec runs on an Ubuntu22 image. That configuration is a property of
+# the codebuild job itself.
+
+# Codebuild's matrix jobs have non-differentiated names so use batch-list
+# instead.
+
+# Parameter motivation
+
+# LIBCRYPTOS
+# awslc: happy path libcrypto for s2n-tls
+# openssl 3: s2n-tls takes different code paths for ossl3, so make sure we run 
+#            asan on it. See pr 4033 for a historical motivating example.
+
+batch:
+  build-list:
+    - identifier: s2nFuzzerAWSLC
```

Comment:
Nit: I like the identifiers that you had in the old job, `clang_awslc` and `clang_openssl_3_0`.

Reply:
I think these look fine for s2nFuzzBatch, but looks a little out of place for the omnibus job since they all have identifier prefixed with `s2n`. I could just make the s2nFuzzBatch using` clang_awslc` and `clang_openssl_3_0` if we don't mind the consistency

Reply:
I change it to using the old identifiers. Omnibus identifiers are defined inside buildspec on CodeBuild so we can name it differently there for consistency